spec/mailers/track_mailer_spec.rb

```diff
@@ -10,6 +10,7 @@
       allow(mail_mock).to receive(:deliver)
       allow(TrackMailer).to receive(:event_digest).and_return(mail_mock)
       allow(Time).to receive(:now).and_return(Time.utc(2007, 11, 12, 23, 59))
+      get_fixtures_xapian_index
```

Comment:
The commit message doesn't explain why we need this here. The specs are currently passing, so not clear on why this is helping.

It does seem to _add_ time to the spec though:

```sh
$ git ls -n 1
b6c38ed Add required xapian setup (Louise Crow)

$ be rspec spec/mailers/track_mailer_spec.rb
Finished in 9.41 seconds (files took 46.89 seconds to load)
20 examples, 0 failures

$ be rspec spec/mailers/track_mailer_spec.rb
Finished in 8.68 seconds (files took 43.73 seconds to load)
20 examples, 0 failures

$ git co HEAD~1
# …
HEAD is now at 8a8e74b... Reduce the number of models used

$ be rspec spec/mailers/track_mailer_spec.rb
Finished in 4.17 seconds (files took 46.07 seconds to load)
20 examples, 0 failures

$ be rspec spec/mailers/track_mailer_spec.rb
Finished in 3.97 seconds (files took 41.44 seconds to load)
20 examples, 0 failures
```